assets/js/components/IngestSheet/List.jsx

```diff
@@ -12,7 +12,7 @@ import { formatDate, TEMP_USER_FRIENDLY_STATUS } from "../../services/helpers";
 
 const IngestSheetList = ({ project, subscribeToIngestSheetStatusChanges }) => {
   const [modalOpen, setModalOpen] = useState(false);
-  const [activeModal, setActiveModal] = useState();
+  const [activeModal, setActiveModal] = useState({});
```

Comment:
In about 99% of cases instead of initializing an empty object literal, it's better to just initial `false`, ie `useState()`.  You'll get into weird scenarios where you're testing against a false value when `{} === false; // is not true`.

Reply:
I edited it to reflect these changes.